include/board.c

```diff
@@ -1,33 +1,69 @@
-#include<stdio.h>
+#include <stdio.h>
 #include "board.h"
 
-void init_board(struct Board *board) {
-    for (int i = 0; i < SIZE; i++) {
-        for (int j = 0; j < SIZE; j++) {
+void init_board(struct Board *board)
+{
+    for (int i = 0; i < SIZE; i++)
+    {
+        for (int j = 0; j < SIZE; j++)
+        {
             board->board[i][j] = EMPTY;
         }
     }
 }
 
-
-void print_board(struct Board *board) {
+void print_board(struct Board *board)
+{
     printf("\n");
     printf("    ");
-    for (int col = 0; col < SIZE; col++) {
+    for (int col = 0; col < SIZE; col++)
+    {
         printf("%d ", col);
     }
     printf("\n");
     printf("    ");
-    for (int col = 0; col < SIZE; col++) {
+    for (int col = 0; col < SIZE; col++)
+    {
         printf("--");
     }
     printf("\n");
-    for (int row = 0; row < SIZE; row++) {
+    for (int row = 0; row < SIZE; row++)
+    {
         printf("%d | ", row);
-        for (int col = 0; col < SIZE; col++) {
+        for (int col = 0; col < SIZE; col++)
+        {
             printf("%c ", board->board[row][col]);
         }
         printf("\n");
     }
     printf("\n");
-}
\ No newline at end of file
+}
+void make_move(struct Board *board)
+{
+    // board->player 1="X"
+    // board->player 2="O"
+
+    if (board->turn % 2 == 0)
+    {
+        board->player = 'X';
+    }
+    else
+    {
+        board->player = 'O';
+    }
+    printf("board->player %c press the number ", board->player);
+    int temp;
+    scanf(" %d", &temp);
```

Comment:
Please keep the functions pure i.e. without any [side effects](https://medium.com/nerd-for-tech/what-are-side-effects-in-programming-51f7ef340f98). `make_move` function should only update the `struct Board` and not take any input or print any output.